.github/workflows/ci.yml

```
@@ -41,55 +32,38 @@ jobs:
       - name: installation of dependencies
         run: npm ci
       - name: start ServeRest locally
-        run: docker run -d -p 3000:3000 paulogoncalvesbh/serverest
+        run: 'docker run -d -p 3000:3000 paulogoncalvesbh/serverest'
       - name: api tests
-        run: npm run cy:run
+        run: 'npm run cy:run'
         env:
           CYPRESS_APP_URL: 'http://localhost:3000'
           CYPRESS_EMAIL: '${{ secrets.EMAIL }}'
           CYPRESS_PASSWORD: '${{ secrets.PASSWORD }}'
-
-  integration-prod:
-    needs: [lint, integration-local]
-    runs-on: ubuntu-18.04
+  allure-report:
+    needs:
+      - integration-prod
+    name: generate allure report
+    runs-on: ubuntu-latest
     steps:
-      - name: checkout
-        uses: actions/checkout@v2
-      - name: node.js 10 setup
-        uses: actions/setup-node@v2
-        with:
-          node-version: 10
-      - name: installation of dependencies
-        run: npm ci
-      - name: api tests
-        run: npm run cy:run
-        env:
-          CYPRESS_APP_URL: 'https://serverest.dev'
-          CYPRESS_EMAIL: '${{ secrets.EMAIL }}'
-          CYPRESS_PASSWORD: '${{ secrets.PASSWORD }}'
-          
-      - name: Get Allure history
+      - uses: actions/checkout@v2
+      - name: get allure history
         uses: actions/checkout@v2
         if: always()
         continue-on-error: true
         with:
           ref: gh-pages
           path: gh-pages
-
-      - name: Allure Report action from marketplace
-        uses: simple-elf/allure-report-action@master
+      - name: allure report action from marketplace
+        uses: ulucasfraga/cypress_api_for_studies@master
         if: always()
-        #id: allure-report
         with:
-          allure_results: build/allure-results
-          #gh_pages: gh-pages
-          #allure_report: allure-report
+          allure_results: allure-results
           allure_history: allure-history
-
-      - name: Deploy report to Github Pages
+          keep_reports: 10
+      - name: deploy report to github pages
         if: always()
         uses: peaceiris/actions-gh-pages@v2
         env:
-          PERSONAL_TOKEN: ${{ secrets.TOKEN }}
+          PERSONAL_TOKEN: '${{ secrets.GH_TOKEN }}'
```

```suggestion
          PERSONAL_TOKEN: ${{ secrets.GH_TOKEN }}
```
Não é preciso o uso dessas aspas

---

thanks!